python/TestHarness/CSVDiffer.py

```diff
@@ -10,7 +10,7 @@
 import os, re, math
 
 class CSVDiffer:
-    def __init__(self, test_dir, out_files, abs_zero=1e-11, relative_error=5.5e-6):
+    def __init__(self, test_dir, out_files, abs_zero=1e-11, relative_error=5.5e-6, gold_dir='./gold'):
```

Comment:
Minor, but we are trying to be better about creating platform neutral python code. There are users that are using this on Windows now. So I'd prefer you to use the "path.join()" utility to create a path without slashes in it for the default above. (os.path.join())

Reply:
Ah, good to know. I was just duplicating what is in the `./run_tests --dump` comment.

Reply:
I only removed the `./`, path.join is os.path.join correct?